eventstore/model_test.go

```diff
@@ -254,7 +255,7 @@ func TestGetInstance(t *testing.T) {
 	t.Run("WithReadTx", func(t *testing.T) {
 		person := &Person{}
 		err = model.ReadTxn(func(txn *Txn) error {
-			txn.FindByID(newPerson.ID, person)
+			_ = txn.FindByID(newPerson.ID, person)
 			checkErr(t, err)
```

Comment:
Wow, your lint fix revealed something that was previously wrong here.
Shouldn't be ignored since the below `checkErr` is doing a wrong check.

Reply:
Will update